proposed/extended-coding-style-guide.md

```diff
@@ -120,42 +120,34 @@ The PHP types and keywords `array`, `int`, `true`, `object`, `float`, `false`, `
 3. Declare Statements, Namespace, and Use Declarations
 --------------------------------------------
 
-When present, there MUST be one blank line after the `declare` statement(s)
-e.g. `declare(ticks=);`
-
-There MUST NOT be a blank line before declare statements such as those for strict
-types or ticks. They MUST be contained on the lines immediately following the
-opening tag (which must be on the first line when declare statement(s) are present).
-
-Each declare statement (e.g. `declare(ticks=);`) MUST be on its own line.
+The header of a PHP file may consist of a number of different blocks.  If present,
+each of the blocks below MUST be separated by a single blank line, and MUST NOT contain
+a blank line.  Each block MUST be in the order listed below, although blocks that are
+not relevant may be omitted.
+
+* Opening PHP tag <?php
+* File-level docblock.
```

Comment:
Do we want to enforce an empty line after the opening php tag and before the file-level docblock as is common practice? This would make sense but an exception to an otherwise clear rule would be a shame.